shopinvader_schema_address/schemas.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from extendable_pydantic import ExtendableModelMeta
+from pydantic import BaseModel
+
+from odoo.addons.pydantic import utils
+
+class Address(BaseModel, metaclass=ExtendableModelMeta):
+    id: int
+    name: str | None
+    street: str | None
+    street2: str | None
+    zip: str | None
+    city: str | None
+    phone: str | None
+    email: str | None
+    state: int | None
+    country: int | None
+    type: str | None
```

Comment:
Is type still used? Since it is not obvious what values it can have, it should be documented with an enum or a Literal type.

Reply:
I will remove it because it is not used. Moreover, the distinction is done by BillingAddress/ShippingAddress..